string state
float32 speed
float32 steer_angle
float32 error1   
float64 error2   
